extensions/datastores/foundationdb/src/main/java/org/locationtech/geowave/datastore/foundationdb/FoundationDBDefaultConfigProvider.java

```diff
@@ -0,0 +1,23 @@
+package org.locationtech.geowave.datastore.foundationdb;
+
+import java.util.Properties;
+import org.locationtech.geowave.core.cli.spi.DefaultConfigProviderSpi;
+
+public class FoundationDBDefaultConfigProvider implements DefaultConfigProviderSpi {
+    private final Properties configProperties = new Properties();
+
+    /**
+     * Create the properties for the config-properties file
+     */
+    private void setProperties() {
+        configProperties.setProperty("store.default-rocksdb.opts.gwNamespace", "default");
```

Comment:
this doesn't belong ... this class can actually be deleted

Reply:
deleted